packages/kokoas-client/src/pages/projInvoice/fieldComponents/EstimatesTableBody.tsx

```diff
@@ -26,11 +30,48 @@ export const EstimateTableBody = ({
   estimateRow: TMaterials
 }) => {
 
+  const navigate = useNavigate();
+
+  const {
+    invoiceId: projInvoiceIdFromURL,
+    custGroupId: custGroupIdFromURL,
+    projEstimateId: estimateIdFromURL,
+  } = useURLParams();
+
+
   const {
     contractAmount,
     billedAmount,
+    estimateId,
   } = estimateRow;
 
+  const handleChange = (_: any, val: boolean | null) => {
```

Comment:
```suggestion
  const handleChange = (_: ChangeEvent<HTMLInputElement>, checked: boolean) => {
```
ここは不要な引数ですが、出来るだけanyを避けたほうがよさそうです。

もう一つのやり方は以下の通りです。

```
const handleChange = (checked: boolean) => { ..// 処理 }

...
onChange = {(_, checked) => handleChange(checked) }
...
```

Reply:
ご指摘ありがとうございます。
コメントで提示して頂いた方が分かりやすく見えたため、そちらで修正させてもらいました。
[```a49433d```](https://github.com/Lorenzras/yumecoco-monorepo/pull/132/commits/a49433d5f5c75e188ca2316d64153e13c661ee7e)にて対応済みです